Pythonwin/pywin/idle/PyParse.py

```diff
@@ -150,7 +150,7 @@ def set_str(self, str):
     # no way to tell the differences between output, >>> etc and
     # user input.  Indeed, IDLE's first output line makes the rest
     # look like it's in an unclosed paren!:
-    # Python 1.5.2 (#0, Apr 13 1999, ...
+    # Python X.X.X (#0, Apr 13 1999, ...
```

Comment:
I wanna say the parenthesis and the fact that IDLE prints some text on opening is what's important to this comment, not the version number (see image, it would seem it's still accurate).
![image](https://github.com/mhammond/pywin32/assets/1350584/a8ad0b42-f1d9-4936-bb6e-8f709ab58eef)

But if you see value in keeping a specific version number in there to know for which Python version this code and comment was initially written, then I can leave it in.

Reply:
oops, yes, I agree